src/libs/Network/RetryCounter.js

```diff
@@ -0,0 +1,27 @@
+export default class RetryCounter {
```

Comment:
Technically this doesn't have to be in the network library. This is a general-purpose JavaScript lib that could be used elsewhere (particularly, there are some GitHub actions that do polling and could conceivably be refactored to use this class).

Reply:
Fair point I can move this.